demisto_sdk/commands/common/tools.py

```diff
@@ -901,11 +901,19 @@ def get_scripts_names(file_path):
         click.secho(f'no scripts found')
     else:
         for script in found_scripts:
-            script_path_full = os.path.join(scripts_dir_path, script, f'{script}.yml')
-            yml_dict = get_yaml(script_path_full)
-            click.secho(f'name: {yml_dict.get("name")}')
-            scripts_names.add(yml_dict.get("name"))
-
+            if script.endswith('.md'):
+                continue  # in case the script is in the old version of CommonScripts - JS code, ignore the md file
```

Comment:
let's try not to use `continue` in the code. If you insist, add a unittest to cover this line.

Reply:
Also, wouldn't we want to add the file name to the known words, even if it ends with .md ?

Reply:
The only option to have .md in a scripts directory is when we have a old version (JS code). In that case, we take care of the yml file (in the next condition).  So, we would like to ignore .md of a script.

Reply:
An assert for the `continue` statement was added to the UT I added